buildimage/uploadrpms

```diff
@@ -37,12 +37,21 @@ tmpcmd=$(mktemp)
 trap "rm -f $tmpcmd" EXIT
 
 (
-    spool=$(mktemp -u "${username}${MYTRAVIS_EP:+@}${MYTRAVIS_EP}.${TRAVIS_BUILD_ID:-XXXX}.XXXXXX");
+    spool=$(mktemp -u "${username}${TARGET_ENDPOINTS:+@}${TARGET_ENDPOINTS}.${BUILD_ID:-XXXX}.XXXXXX");
     echo "mkdir $spool";
     echo "cd $spool";
-    for F in $(find . -name '*.rpm'); do
-        echo "put $F";
-    done;
+    if [[ "$GITHUB_ACTIONS" && "$GPG_SIGN_KEY" ]]; then
+        echo "$GPG_SIGN_KEY" | gpg --import &> /dev/null
+        while IFS= read -r -d '' file
+        do
+            gpg --encrypt --yes --trust-model always --user nethinfra@nethesis.it "$file" &> /dev/null
```

Comment:
Stderr might contain useful information to understand and debug this. I'd send it to the default descriptor (build log).
```suggestion
        echo "$GPG_SIGN_KEY" | gpg --import > /dev/null
        while IFS= read -r -d '' file
        do
            gpg --encrypt --yes --trust-model always --user nethinfra@nethesis.it "$file" > /dev/null
```